src/docs/Components/Form/FileUploader/index.mdx

```diff
@@ -26,18 +62,22 @@ After uploading a file, a tile is added to the bottom of the component. This til
 - Uploaded
 - Error
 
-<Preview path="file-uploader-with-file" nude />
+<Preview path="uploaded-files" nude />
```

Comment:
aren't we supposed to have an error message displayed for the `filename-invalid.jpg`?

Reply:
@nherwegh Yes you're right. I will fix the preview ;)